Automattic/simplenote-macos
# Published Icon

### Details:
In this PR we're adding a "Published" Indicator in the right hand side of any note that has been made publicly available.

Closes #4 

cc @bummytime for code review
cc @SylvesterWilmott for design review!

Thank you both!!


### Testing:
1. Log into your account
2. Select any of your notes
3. Click over the Share Icon (Top Right actions, third one from the right)
4. Select `Publish to Web`
5. Publish the note!

Verify that published notes get an icon in the right hand side of their cells.

---

### Screenshots:

<img width="1219" alt="Screen Shot 2019-07-04 at 2 36 14 PM" src="https://user-images.githubusercontent.com/1195260/60682282-228e0c00-9e69-11e9-98ab-d36eab315964.png">
<img width="1263" alt="Screen Shot 2019-07-04 at 2 36 24 PM" src="https://user-images.githubusercontent.com/1195260/60682284-2326a280-9e69-11e9-8d76-9e975f6129ae.png">
